audio/rsgain/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                      1.0
+
+PortGroup                       github 1.0
+PortGroup                       cmake 1.1
+
+github.setup                    complexlogic rsgain 3.5.2 v
+
+categories                      audio
+platforms                       darwin freebsd linux netbsd openbsd
+license                         GPL-2+
+
+maintainers                     {gmail.com:lmkrawiec @lmkra}
+
+description                     Really simple gain
+long_description                ReplayGain 2.0 command line utility \
+                                rsgain applies loudness metadata tags to your files, \
+                                while leaving the audio stream untouched. \
+                                A ReplayGain-compatible player will dynamically adjust \
+                                the volume of your tagged files during playback.
+
+homepage                        https://github.com/complexlogic/rsgain
```

Comment:
This `homepage` line isn't needed, since it's implicitly set by `github.setup`, and can be dropped.